assets/queries/terraform/aws/vpc_subnet_assigns_public_ip/test/positive_expected_result.json

```
@@ -0,0 +1,7 @@
+[
+  {
+    "queryName": "VPC Subnet Assigns Public IP",
+    "severity": "MEDIUM",
+    "line": 13
```

Good practice for our test framework to add `fileName`
```suggestion
    "line": 13,
    "fileName": "positive.tf"
```